addons/ai/XEH_preInit.sqf

```diff
@@ -4,4 +4,16 @@ ADDON = false;
 
 #include "XEH_PREP.hpp"
 
+[QGVAR(updateActiveGroups), {
+    params ["_newGroups"];
+    private _spawnedGroups = GVAR(activeGroups) getOrDefault [_id, []];
+    _spawnedGroups append _newGroups;
+    GVAR(activeGroups) set [_id, _spawnedGroups];
+}] call CBA_fnc_addEventHandler;
```

Comment:
Wrap in an `if (isServer)` check

Reply:
Do you literally mean wrap or can I do exitWith?

Reply:
literal wrap with an if then. only server needs this